data/json/items/fluff.json

```diff
@@ -98,143 +112,1384 @@
+    "weight": "680 g",
+    "volume": "1250 ml",
+    "price": 12950,
+    "price_postapoc": 500,
+    "material": [ "plastic" ],
+    "symbol": "?",
+    "color": "red",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "dnd_3",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Sticks and Stones", "str_pl": "sets of Sticks and Stones" },
+    "description": "A roleplaying game set in the Dawn of Man, the Primitive Age!  There, you can choose different places in the world to start, select many different classes, and different species of hominids to survive the primitive world.",
+    "weight": "680 g",
+    "volume": "1250 ml",
+    "price": 12950,
+    "price_postapoc": 500,
+    "material": [ "plastic" ],
+    "symbol": "?",
+    "color": "red",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "g_warhammer",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Battlehammer", "str_pl": "sets of Battlehammer" },
+    "description": "A strategy game featuring a set of tiny figurines of fantasy creatures.",
+    "weight": "680 g",
+    "volume": "1250 ml",
+    "price": 10880,
+    "price_postapoc": 500,
+    "material": [ "plastic" ],
+    "symbol": "?",
+    "color": "yellow",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "g_warhammer40k",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Battlehammer 20k", "str_pl": "sets of Battlehammer 20k" },
+    "description": "A strategy game featuring a set of tiny figurines of space aliens and grotesque space marines.",
+    "weight": "680 g",
+    "volume": "1250 ml",
+    "price": 10880,
+    "price_postapoc": 500,
+    "material": [ "plastic" ],
+    "symbol": "?",
+    "color": "yellow",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "catan",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Settlers of the Ranch", "str_pl": "sets of Settlers of the Ranch" },
+    "description": "A strategy game where players build settlements and trade for supplies.",
+    "weight": "804 g",
+    "volume": "1250 ml",
+    "price": 7050,
+    "price_postapoc": 500,
+    "material": [ "wood" ],
+    "symbol": "?",
+    "color": "light_blue",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "agricola",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Farmland", "str_pl": "sets of Farmland" },
+    "description": "A fun strategy board game where players (or just a player, since you can play this alone as a solitaire board game) attempt to improve their farms by plowing fields, sowing crops, raising livestock, and expanding their families.  The game is set in the European farming community during the 17th century..",
```

Comment:
```suggestion
    "description": "A fun strategy board game where players (or just a player, since you can play this alone as a solitaire board game) attempt to improve their farms by plowing fields, sowing crops, raising livestock, and expanding their families.  The game is set in the European farming community during the 17th century.",
```